tests_and_analysis/test/script_tests/test_powder_map.py

```diff
@@ -0,0 +1,125 @@
+import os
+import json
+from unittest.mock import patch
+
+import pytest
+import numpy.testing as npt
+# Required for mocking
+import matplotlib.pyplot
+
+from tests_and_analysis.test.utils import get_data_path, get_castep_path, get_phonopy_path
+from tests_and_analysis.test.script_tests.utils import (
+    get_script_test_data_path, get_current_plot_image_data, args_to_key)
+import euphonic.cli.powder_map
+
+graphite_fc_file = get_castep_path('graphite', 'graphite.castep_bin')
+nacl_prim_fc_file = get_phonopy_path('NaCl_prim', 'phonopy_nacl.yaml')
+powder_map_output_file = os.path.join(get_script_test_data_path(),
+                                      'powder-map.json')
+
+quick_calc_params = ['--npts=10', '--npts-min=10', '--q-spacing=1']
+powder_map_params = [
+    [nacl_prim_fc_file],
+    [nacl_prim_fc_file, *quick_calc_params],
+    [nacl_prim_fc_file, '--temperature=1000', *quick_calc_params],
```

Comment:
Commenting before I forget, there should be a combination of `--temperature` and `-w coherent` or the temperature actually doesn't do anything...

Reply:
done!